lib/elixir/lib/registry.ex

```diff
@@ -209,10 +225,22 @@ defmodule Registry do
     end
   end
 
+  @doc false
+  @doc since: "1.x.x"
+  def whereis_name({registry, key, _value}) do
+    whereis_name({registry, key})
```

Comment:
I'd prefer if we had helper functions called from both callbacks, for example:

```
def whereis_name({registry, key, _value}), do: whereis_name(registry, key)
def whereis_name({registry, key}), do: whereis_name(registry, key)

defp whereis_name(registry, key) do
```

Reply:
Great point.  Congrats on joining the core team, btw!